lumen/ai/agents.py

```diff
@@ -504,14 +521,30 @@ async def retry_invoke(event: param.parameterized.Event):
 
 class SQLAgent(LumenBaseAgent):
 
+    conditions = param.List(
+        default=[
+            "Preferred for data queries requiring table joins, filtering, or new calculations",
+            "Better choice than TableListAgent when asked to show table data",
+            "Works with IterativeTableLookup when additional columns are needed",
+            "Only use for new data analysis; for visualization of existing results, use current data",
+            "Defer to DbtslAgent when dbtsl_metaset is available",
```

Comment:
Only comment on this PR is that I think the more generic agent shouldn't be listing conditions about a more specific agent, i.e. I'd invert this and have DbtAgent have a condition that says: "Prefer DbtslAgent if dbtsl_metaset" is available.